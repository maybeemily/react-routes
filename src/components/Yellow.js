import React from 'react';

export default function Yellow() {
  const style = {
    backgroundColor: 'yellow',
    width: '50vw',
    height: '50vw',
    fontSize: '4em',
    color: 'black'
  };

  return (
    <div style={style}>Y E L L O W</div>
  );
}
