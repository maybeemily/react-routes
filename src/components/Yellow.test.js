import React from 'react';
import { shallow } from 'enzyme';
import Yellow from './Yellow';

describe('Header component', () => {
  it('renders the header', () => {
    const wrapper = shallow(<Yellow/>);
    expect(wrapper).toMatchSnapshot();
  });
});
