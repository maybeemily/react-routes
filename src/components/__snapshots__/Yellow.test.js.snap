// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the header 1`] = `
<div
  style={
    Object {
      "backgroundColor": "yellow",
      "color": "black",
      "fontSize": "4em",
      "height": "50vw",
      "width": "50vw",
    }
  }
>
  Y E L L O W
</div>
`;
